### ISSUE TITLE
Add Definitions/Descriptions

### ISSUE THREAD

**Issue by katieefrey** [astrothesaurus/UAT#314]
Push to add descriptions/definitions for many concepts for the 2020 Dec Release.  May not get a description for every concept by release date, but starting to make a concentrated effort to add them.

See current definitions progress here: https://docs.google.com/spreadsheets/d/1gCDNWnkpFIkS1qNz9GGHhfQ_Tr2H623-vJ9QkS9v3kg/edit?usp=sharing

Please add comments/suggestions to this thread.

**Comment by katieefrey** [comment]
Was able to match 388 UAT concepts with concepts from https://dictionary.obspm.fr using a python script. I believe that additional similar matches exist which I will probably need to identify by hand.

Concepts with an exact match listed here to aid searching:

absolute  magnitude
accelerating universe
accretion
active sun
aeronomy
amateur astronomy
angular separation
annual equation
annual parallax
anomalistic month
antapex
anthropic principle
antumbra
apastron
aperture synthesis
apex
aphelion
apocenter
apogee
apparent magnitude
apparent position
apsidal motion
archaeoastronomy
argument of perigee
argument of perihelion
ascending node
asteroid belt
asteroseismology
astrobiology
astrochemistry
astrodynamics
astrometry
astrophotography
atmospheric circulation
atmospheric extinction
atmospheric refraction
autumnal equinox
azimuth
baade-wesselink method
baily's beads
baryogenesis
baryonic dark matter
bayes' theorem
big bang theory
blazhko effect
blue loop
bode's law
bolometric correction
brightness temperature
carbon burning
carbon dioxide
caustic curve
celestial equator
celestial mechanics
celestial sphere
center of mass
ceres
chandrasekhar limit
chaos
chemical enrichment
circumstellar dust
circumstellar matter
classification
closed universe
clustering
collimation
coma cluster
compact h ii region
conjunction
cosmic background radiation
cosmic rays
cosmic web
cosmochemistry
cosmogony
cosmological constant
cosmology
dark energy
dark matter
de sitter universe
deceleration parameter
declination
deconvolution
degenerate matter
descending node
detection
diamond ring effect
diurnal parallax
doppler shift
dredge-up
dynamical friction
dynamical parallax
early universe
earth-moon system
eccentric anomaly
eccentricity
ecliptic latitude
ecliptic longitude
effective temperature
e-folding time
ejecta
equatorial coordinate system
equinox
ergosphere
eris
ethnoastronomy
exosphere
expanding universe
explosive nucleosynthesis
extinction
extragalactic astronomy
field of view
flux calibration
galactic anticenter
galactic bulge
galactic center
galactic latitude
galactic longitude
galaxy formation
galilean satellites
gamma-ray astronomy
gas-to-dust ratio
gegenschein
general relativity
geocorona
giant branch
gravitation
gravitational collapse
gravitational energy
gravitational equilibrium
gravitational instability
gravitational interaction
gravitational lensing
gravity darkening
great attractor
greenhouse gases
haumea
hawking radiation
hayashi track
heliopause
helioseismology
heliosheath
heliosphere
helium burning
henyey track
hertzsprung gap
hierarchical cosmology
horizontal parallax
hot dark matter
hour angle
hubble constant
hubble diagram
hubble radius
hubble time
humidity
hydrodynamics
hydrogen burning
hydrosphere
hyperbolic orbit
inclination
infrared astronomy
infrared excess
inner planets
instability strip
integrated magnitude
interacting galaxies
intercloud medium
interferometry
intergalactic medium
interplanetary dust
interplanetary gas
interplanetary medium
interstellar absorption
interstellar dust
interstellar extinction
interstellar radiation field
interstellar reddening
interstellar scintillation
ionization
ionosphere
jeans mass
jupiter
kappa mechanism
keplerian orbit
kruskal diagram
kuiper belt
libration
light year
limb brightening
limb darkening
limiting magnitude
line of sight
linear regression
lithosphere
local group
lomb-scargle periodogram
luminosity function
lunar maria
lunar phase
lyman alpha forest
magellanic clouds
magellanic stream
magnitude
main sequence
major axis
makemake
mantle
mars
mascon
mass spectrometry
maunder minimum
mean anomaly
meridian
mesosphere
metallicity
methane
microgravity
milky way galaxy
minor axis
missing mass
morgan-keenan classification
nadir
n-body problem
neptune
newtonian cosmology
normal distribution
north celestial pole
nuclear physics
nucleosynthesis
oblateness
observational cosmology
occultation
oort cloud
open universe
opposition
oscillating universe
osculating orbit
oxygen burning
ozone layer
parabolic orbit
parallax
particle physics
penrose process
penumbra
periapsis
periastron
perigee
perihelion
perseus cluster
phase angle
photoelectric magnitude
photoelectric photometry
photographic magnitude
photoionization
photometer
photometric parallax
photometry
photon sphere
photovisual magnitude
planetary science
plasma physics
plasmapause
plate tectonics
pluto
poisson distribution
polar orbit
polarimetry
position angle
proton-proton reaction
purkinje effect
quantum gravity
quaoar
quiet sun
quintessence
radial velocity
radiative braking
radiative recombination
radiative transfer
radiative transfer equation
radio astronomy
radio continuum emission
rapidly oscillating ap star
recombination
regression
reionization
relativistic aberration
relativity
relaxation time
right ascension
roche limit
roche lobe
r-process
saturn
scale height
schwarzschild metric
schwarzschild radius
sculptor dwarf elliptical   galaxy
secondary cosmic rays
secular parallax
sedna
shadow bands
shell burning
silicon burning
sky brightness
smoothing
solar activity
solar constant
solar corona
solar cycle
solar dynamo
solar mass
solar neighborhood
solar neutrino problem
solar parallax
solar physics
solar radiation
solar radius
solar rotation
solar system
solar wind
solstice
south celestial pole
space debris
space weather
special relativity
speckle interferometry
spectral index
spectrophotometry
spectropolarimetry
spectroscopy
s-process
standard stars
star formation
statistical parallax
stellar astronomy
stellar dynamics
stellar evolution
stellar feedback
stellar nucleosynthesis
stellar parallax
stellar photometry
stellar physics
stellar rotation
stratosphere
strong gravitational lensing
submillimeter astronomy
summer solstice
sunspot cycle
sunspot number
supergalactic coordinate   system
supergalactic latitude
supergalactic longitude
surface gravity
synodic period
syzygy
termination shock
thermosphere
three-body problem
tidal disruption
tidal friction
tidal radius
trigonometric parallax
tropopause
troposphere
true anomaly
true position
turnoff point
twilight
two-body problem
ultraviolet astronomy
umbra
upper atmosphere
uranus
urca process
venus
vernal equinox
vertical circle
virgo cluster
virgo supercluster
visibility function
von zeipel theorem
walraven photometry
warm-hot intergalactic medium
water vapor
weak gravitational lensing
wilson effect
winter solstice
x-ray astronomy
zenith

[uat_definitions-matches.xlsx](https://github.com/astrothesaurus/UAT/files/4771739/uat_definitions-matches.xlsx)

**Comment by katieefrey** [comment]
421 concepts now have definitions.  This is just about exactly 20% of the UAT.  All newly added definitions have been sourced from the Etymological Dictionary of Astronomy and Astrophysics, there may have been a few concepts that already had definitions from other sources.

These new definitions represent the ones with exact matches between UAT and EDAA, plus another 30-40 that I have found through concept searching to account for things like plurals.  I'm still not finished mining EDAA for concept definitions as I am sure there are still many many near (but not exact) matches that I can use.  This comment is meant to be a status update.

Updated UAT/EDAA matches spreadsheet.
[uat_definitions-matches.xlsx](https://github.com/astrothesaurus/UAT/files/5024454/uat_definitions-matches.xlsx)

Current UAT definitions export:
[uat_list_with_defintions2020_0804_1632.xlsx](https://github.com/astrothesaurus/UAT/files/5024461/uat_list_with_defintions2020_0804_1632.xlsx)

Note, I didn't take care to handle UTF-8 characters properly in the files for this progress update.

All 421 concepts with definitions so far are listed here to aid future searching:
Absolute magnitude, Accelerating universe, Active sun, Aeronomy, Alt-azimuth mounts, Am stars, Amateur astronomy, Annual equation, Annual parallax, Anomalistic month, Antapex, Anthropic principle, Antumbra, Ap stars, Apastron, Aperture synthesis, Apex, Aphelion, Apocenter, Apogee, Apparent magnitude, Apsidal motion, Archaeoastronomy, Argument of perigee, Argument of perihelion, Ascending node, Asteroid belt, Asteroseismology, Astrobiology, Astrochemistry, Astrodynamics, Astrometry, Astrophotography, Atmospheric circulation, Atmospheric extinction, Atmospheric refraction, Autumnal equinox, Azimuth, B(e) stars, Baade-Wesselink method, Baily's beads, Baryogenesis, Baryonic dark matter, Bayes' Theorem, Big Bang theory, Blazhko effect, Blue loop, Bolometric correction, Brightness temperature, Carbon burning, Carbon dioxide, Caustic curve, Celestial equator, Celestial mechanics, Celestial sphere, Ceres, Chandrasekhar limit, Chemical enrichment, Chemically peculiar stars, Circumstellar dust, Circumstellar matter, Classification, Closed universe, Clustering, Cold dark matter, Collimation, Coma Cluster, Comae, Compact H II region, Conjunction, Cosmic rays, Cosmic web, Cosmochemistry, Cosmogony, Cosmological constant, Cosmology, Dark adaption, Dark energy, Dark matter, de Sitter universe, Deceleration parameter, Declination, Deconvolution, Degenerate matter, Descending node, Detection, Diamond ring effect, Diurnal parallax, Domes, Doppler shift, DQ Herculis stars, Dredge-up, Dynamical friction, Dynamical parallax, Early universe, Earth (planet), Earth-moon system, Eccentric anomaly, Eccentricity, Ecliptic latitude, Ecliptic longitude, Effective temperature, E-folding time, Ejecta, Ephemerides, Equatorial coordinate system, Equinox, Ergosphere, Eris, Ethnoastronomy, Exoplanet systems, Exoplanets, Exosphere, Expanding universe, Explosive nucleosynthesis, Extinction, Extragalactic astronomy, Fanaroff-Riley radio galaxies, Field galaxies, Field of view, Field stars, Flux calibration, Galactic anticenter, Galactic bulge, Galactic center, Galactic latitude, Galactic longitude, Galaxy dark matter halos, Galaxy formation, Galilean satellites, Gamma-ray astronomy, Gas-to-dust ratio, Gegenschein, General relativity, Geocorona, Geodesics, Giant branch, Gravitation, Gravitational collapse, Gravitational equilibrium, Gravitational instability, Gravitational interaction, Gravitational lensing, Gravity darkening, Great Attractor, Greenhouse gases, Ground-based astronomy, Haumea, Hawking radiation, Hayashi track, Heliopause, Helioseismology, Heliosheath, Heliosphere, Helium burning, Helium weak stars, Helium-poor stars, Henry Draper catalog, Henyey track, Hertzsprung gap, Hertzsprung Russell diagram, Hierarchical cosmology, Horizontal parallax, Hot dark matter, Hot Jupiters, Hour angle, Hubble classification scheme, Hubble constant, Hubble diagram, Hubble radius, Hubble time, Humidity, Hydrodynamics, Hydrosphere, Hyperbolic orbit, Inclination, Infrared astronomy, Infrared excess, Inner planets, Instability strip, Integrated magnitude, Interacting galaxies, Intercloud medium, Interferometry, Intergalactic medium, Interplanetary dust, Interplanetary gas, Interplanetary medium, Interstellar absorption, Interstellar dust, Interstellar extinction, Interstellar medium wind, Interstellar radiation field, Interstellar reddening, Interstellar scintillation, Ionization, Ionosphere, Jeans mass, Jupiter, Jupiter trojans, Kappa mechanism, Keplerian orbit, Kruskal diagram, Kuiper belt, L galaxies, Libration, Light year, Limb brightening, Limb darkening, Limiting magnitude, Line of sight, Linear regression, Lithosphere, Local Group, Lomb-Scargle periodogram, Longitude of the ascending node, Luminosity function, Lunar craters, Lunar domes, Lunar highlands, Lunar maria, Lunar phase, Lunar rilles, Lyman alpha forest, Magellanic Clouds, Magellanic Stream, Magnetohydrodynamics, Magnitude, Main sequence, Major axis, Makemake, Mantle, Mars, Mars trojans, Mascon, Mass spectrometry, Maunder minimum, Mean anomaly, Mercury (planet), Mercury-manganese stars, Meridian, Meridian circles, Mesosphere, Metallicity, Methane, Microgravity, Minor axis, Missing mass, Nadir, Natural satellites (Extrasolar), Natural satellites (Solar system), N-body problem, Neptune, Newtonian cosmology, Normal distribution, North celestial pole, Nuclear physics, Nucleosynthesis, Oblateness, Observational cosmology, Occultation, Oort cloud, Open universe, Opposition, Orbital elements, Oscillating universe, Osculating orbit, Oxygen burning, Ozone layer, Parabolic orbit, Parallax, Particle physics, Penrose process, Penumbra, Periapsis, Periastron, Perigee, Perihelion, Perseus Cluster, Phase angle, Photoelectric magnitude, Photoelectric photometry, Photographic magnitude, Photoionization, Photometer, Photometric parallax, Photometry, Photon sphere, Photovisual magnitude, Planetary science, Plasma physics, Plasmapause, Plate tectonics, Pluto, Poisson distribution, Polar orbit, Polarimetry, Position angle, Pre-main sequence stars, Proton-proton reaction, Purkinje effect, Quantum gravity, Quaoar, Quiet sun, Quintessence, Radial velocity, Radiative braking, Radiative recombination, Radiative transfer, Radiative transfer equation, Radio astronomy, Radio continuum emission, Rapidly oscillating Ap star, Recombination, Regression, Reionization, Relativistic aberration, Relaxation time, Right ascension, Roche limit, Roche lobe, R-process, Saturn, Scale height, Schonberg-Chandrasekhar limit, Schwarzschild metric, Schwarzschild radius, Sculptor dwarf elliptical galaxy, Secondary cosmic rays, Secular parallax, Sedna, Shadow bands, Shell burning, Silicon burning, Sky brightness, Smoothing, Solar activity, Solar constant, Solar corona, Solar cycle, Solar dynamo, Solar faculae, Solar mass, Solar neighborhood, Solar neutrino problem, Solar parallax, Solar physics, Solar radiation, Solar radius, Solar rotation, Solar system, Solar wind, Solstice, South celestial pole, Space debris, Space weather, Special relativity, Speckle interferometry, Spectrophotometry, Spectropolarimetry, Spectroscopy, S-process, Standard stars, Star formation, Statistical parallax, Stellar astronomy, Stellar dynamics, Stellar evolution, Stellar feedback, Stellar nucleosynthesis, Stellar occultation, Stellar parallax, Stellar photometry, Stellar physics, Stellar rotation, Stratosphere, Strong gravitational lensing, Submillimeter astronomy, Summer solstice, Sunspot number, Supergalactic coordinate system, Surface gravity, Synodic period, Syzygy, Termination shock, The Moon, The Sun, Thermosphere, Three-body problem, Tidal disruption, Tidal friction, Trigonometric parallax, Tropopause, Troposphere, True anomaly, True position, Turnoff point, Twilight, Two-body problem, Ultraviolet astronomy, Umbra, Upper atmosphere, Uranus, Urca process, Venus, Vernal equinox, Vertical circle, Very long baseline interferometry, Virgo Cluster, Virgo Supercluster, Visibility function, von Zeipel theorem, Walraven photometry, Warm-hot intergalactic medium, Water vapor, Weak gravitational lensing, Wilson effect, Winter solstice, Xenobiology, X-ray astronomy, Zenith
